### DISCUSSION TITLE
Image elements do not have explicit width and height

### DISCUSSION THREAD

**Discussion by Elindo586** [vercel/next.js#40186] [Help]
So I am getting this error below... but why? 

![img33](https://user-images.githubusercontent.com/93617433/188224983-fabd35ea-3523-4a90-80c8-6429b8c2d122.PNG)

All of my images showing on the browser are png and with a code that looks like this:

```
<div id="maintenance1">
            <Image
              src="/images/maintenance-1.png"
              id="maintenance1"
              alt="maintenance one pic"
              width={700}
              height={400}
            />
          </div>

```

^^^  this is a sample code,  pictures might have different sizes, but would follow the same code...

According to lighthouse it doesn't show explicit width and height, but isn't already there on the next img tag ???

**Comment by nagayev** [comment]
You can try to use usual html `img` tag.
I think problem with style attribute: 
`width: initial, height: initial`
So your code should be
```jsx
<img
              src="/images/maintenance-1.png"
              id="maintenance1"
              alt="maintenance one pic"
              width={700}
              height={400}
            />
```

**Comment by jakejarvis** [comment]
Definitely don't switch to `<img>` tags, you're doing everything correctly!

It's actually related to the blurry SVG placeholders, not the actual image, and according to https://github.com/vercel/next.js/issues/39376#issuecomment-1229201173 will be fixed in the next version of Lighthouse. Nothing to do except wait for that, unfortunately. :)

**Comment by icyJoseph** [comment]
This is a sort of funny thing with lightouse, because the next/image developed by Google doesn't really agree with the lightouse reports made also by Google :) - 

Take inspection tools results with a grain of salt, it is at the end of the day a product that's always improving and it is full of best practices and opinions that change over time. Stay with the next/image, you don't have to do as the tool says if you don't see or experience any actual issues for you app, moreover if there's documentation indicating that this is an issue on the measurement tool end.
